lib/webidl2.js

```diff
@@ -174,37 +174,14 @@
       "line-comment": /\/\/(.*)\r?\n?/y,
       "multiline-comment": /\/\*((?:[^*]|\*[^/])*)\*\//y
     };
-    function all_ws(store, pea) { // pea == post extended attribute, tpea = same for types
+    function all_ws(pea) { // pea == post extended attribute, tpea = same for types
```

Comment:
At some point, we should consider renaming these things properly. There is no reason to have such short argument and variable names. Will make this code much more maintainable and approachable.

Reply:
Ooops, actually that `pea` is also should be removed as it's part of opt.ws.